ui/oopsyraidsy/player_state_tracker.ts

```diff
@@ -486,7 +490,7 @@ export class PlayerStateTracker {
 
   OnHoTDoT(_line: string, splitLine: string[]): void {
     const targetId = splitLine[logDefinitions.NetworkDoT.fields.id];
-    if (!targetId || !this.IsInParty(targetId))
+    if (targetId === undefined || targetId.length === 0 || !this.IsInParty(targetId))
```

Comment:
```suggestion
    if (targetId === undefined || !this.IsInParty(targetId))
```

`IsInParty` covers this one too.